flottform/forms/src/flottform-channel-host.ts

```diff
@@ -219,6 +223,25 @@ export class FlottformChannelHost extends EventEmitter<FlottformEventMap> {
 		return response.json();
 	};
 
+	private fetchIceServers = async (baseApi: string) => {
+		const response = await fetch(`${baseApi}/turn-credentials`, {
```

Comment:
We're overwriting the full `iceServers`, so maybe we want to use `ice-server-credentials` instead of `turn`? They include STUN server as well, right?

Reply:
Yes the response contains STUN & TURN servers if `useTurnServer=true`! I'll change `turn-credentials` to `ice-server-credentials`.